src/Utilities/FileSystem.cpp

```diff
@@ -78,35 +78,33 @@ std::string get_absolute_path(const std::string& rel_path) {
           << rel_path << "'.");
       // LCOV_EXCL_STOP
     }
-    ERROR("Failed to get absolute path for an unknown reason.");
+    ERROR("Failed to get absolute path for an unknown reason: " << errno);
```

Comment:
Is it safe to use `errno` inside `ERROR`?  I don't know what guarantees the standard library stuff gives about errno.